### ISSUE TITLE
Pod is stuck in Init:CrashLoopBackOff

### ISSUE THREAD

**Issue by kumarallamraju** [kelseyhightower/certificate-init-container#13]
kubectl create -f tls-app.yaml

$ kubectl get pods
NAME                      READY     STATUS       RESTARTS   AGE
tls-app-dc7787455-69rhx   0/1       Init:Error   1          6s
$ kubectl get pods
NAME                      READY     STATUS                  RESTARTS   AGE
tls-app-dc7787455-69rhx   0/1       Init:CrashLoopBackOff   1          12s
$ kubectl logs tls-app-dc7787455-69rhx
Error from server (BadRequest): container "tls-app" in pod "tls-app-dc7787455-69rhx" is waiting to start: PodInitializing

I'm just running the yaml uploaded in keley's github repository. Am I missing something?

**Comment by hgsat123** [comment]
Yes. This is not working. I tried it on GCP.  Getting "crash-back off" error
tls-app-77cf5bf94c-qxphr                                     0/1       Init:CrashLoopBackOff   6          9m

# kubectl logs -f tls-app-77cf5bf94c-qxphr --namespace kube-system
Error from server (BadRequest): container "tls-app" in pod "tls-app-77cf5bf94c-qxphr" is waiting to start: PodInitializing

**Comment by jbornemann** [comment]
Its possible you are running into https://github.com/kelseyhightower/certificate-init-container/issues/12 . 

```
kubectl logs <pod> -n <namespace> -c certificate-init-container
```
specifying the init container, to know for sure. Once I made some changes in my fork, per my version of k8s, I was golden.
